app/views/devise/sessions/new.html.erb

```diff
@@ -2,4 +2,24 @@
 
 <h1><%= t '.header' %></h1>
 <p><%= t '.description_html' %></p>
-<%= link_to t('.login_button_text'), user_shibboleth_omniauth_authorize_path, :class => "btn btn-primary" %>
+<%= link_to t('.login_button_text'), user_shibboleth_omniauth_authorize_path, class: 'btn btn-primary' %>
+
+<%- if Rails.env.development? %>
+  <%# Temporary login form for local development until IST shibboleth issues gets resolved %>
```

Comment:
Until we fix the bottleneck of shibboleth on local machines and get something more practical you can still use the login form in development environments. This will simplify development for time being.

You can simply just create a user from the console like so:
`$ rails c`
`> User.create(email: 'test@email.com', password: 'password', provider: 'email', uid: '12345678')`

Then login with this user with whatever you used for email/password. in this provided login form.